variables.tf

```diff
@@ -417,3 +417,9 @@ variable "cloudfront_origin_access_identity_path" {
   default     = ""
   description = "Existing cloudfront origin access identity path used in the cloudfront distribution's s3_origin_config content"
 }
+
+variable "origin_public_access_block" {
```

Comment:
@juliusdanielherreraglomm can you update this variable name to `block_origin_public_access_enabled` or something similar? We use the `_enabled` convention to denote `bool` var where we can.

Reply:
Good idea. Done.

Reply:
Hi @Gowiem

Happy new year!

I tried include your suggestion and rebased it. I hope that this is now ready to be merged.
Let me know if I can do more.

Cheers

Julius